data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
We have miscommunication here. Fixing of pantss is reasonable because pants are actually  stacking in game and its incorrect plural name is shown in game. Meanwhile, it's not possible to see plural form of name for items stored by charges, so fix for fungicide is not bad but useless.